instrumentation/netty-reactor-0.9.0/src/main/java/reactor/netty/http/server/HttpTrafficHandler_Instrumentation.java

```diff
@@ -0,0 +1,27 @@
+/*
+ *
+ *  * Copyright 2020 New Relic Corporation. All rights reserved.
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
```

Comment:
This header needs to be added to all source files.

Reply:
Does it? Some of this code was based off open telemetry instrumentation (as called out in comments and the `NOTICE.txt` in the root of these modules.  Was the omission of these headers in the `0.8.0` module accidental or intentional?

Reply:
It was a bit of both, I'll add the headers to the classes that aren't derived from Otel